runtime/mainnet/src/config/governance.rs

```diff
@@ -0,0 +1,387 @@
+use frame_support::{
+	parameter_types,
+	traits::{EitherOfDiverse, NeverEnsureOrigin},
+	weights::Weight,
+};
+use frame_system::EnsureRoot;
+use pallet_collective::{EnsureMembers, EnsureProportionAtLeast};
+use parachains_common::BlockNumber;
+use pop_runtime_common::DAYS;
+
+use crate::{
+	AccountId, Council, Runtime, RuntimeBlockWeights, RuntimeCall, RuntimeEvent, RuntimeOrigin,
+};
+
+// Type aliases for council origins.
+type AllMembersButOne = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	// Currently we set 4 members at genesis.
+	EnsureMembers<AccountId, CouncilCollective, 3>,
```

Comment:
what happens if we add or remove a new member. Wouldn't this break?

Reply:
Exactly, I'm not specially happy with that option because 3 will only be every member minus one under that concrete condition.

I guess whether this is useful or not depends on how we foresee the council to change in the short term.

I'd be more comfortable merging the at least proportion than this fixed amount of members.